examples/plotting/custom_tooltip.py

```diff
@@ -1,3 +1,15 @@
+''' A plot of periodic table elements using a Periodic table dataset.
+This example demonstrates the use of custom css tooltips when creating plots.
+The chart shows correlation between atomic mass and density of gssperiodic table elements.
+
+.. bokeh-example-metadata::
+    :sampledata: periodic_table.elements
+    :apis: bokeh.plotting.figure
+    :refs: :ref:`userguide_plotting` > :ref:`ug_interaction_tooltips`
```

Comment:
We stopped doing the manual "breadcrumbs" just too difficult to maintain. 
```suggestion
    :refs: :ref:`ug_interaction_tooltips`
```